lib/kira/projects/services/find_task_to_assign.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Kira.Projects.Services.FindTaskToAssign do
+  @moduledoc """
+  Returns an open task with no assignee.
+  """
+
+  use Exop.Operation
+
+  alias Kira.Projects.Entities.Issue
+  alias Kira.Projects.Queries.IssueQueries
+  alias Kira.Projects.Queries.ProjectQueries
+
+  parameter(:project_uid, type: :integer)
+
+  def process(%{project_uid: project_uid}) do
+    project = ProjectQueries.get_project!(project_uid)
+
+    issue =
+      Issue
+      |> IssueQueries.for_project(project)
```

Comment:
There's also one more step here. We need to find `queued` state tasks. Please, create a new task.

Reply:
Do you mean there's a state `queued` for issues? So we have to find both `opened` and `queued`

Reply:
Just `queued`

Reply:
I see, just opened issues are not going to be assigned, makes sense. Thanks